### DISCUSSION TITLE
I would like to know if the BSP -chop command is subdividing the triangles with different sizes.

### DISCUSSION THREAD

**Discussion by JafiMapping** [ericwa/ericw-tools#380] [Q&A] [answered]
Hello! I would like to know if the BSP -chop command is subdividing the triangles with different sizes. It was useful to make bigger outside maps.  Btw, i'm mapping for the Quake2ex.   -chop 1024. for an exemple.  Btw, i'm mapping for the Quake2ex. Thank you.

**Comment by Paril** [comment]
I don't think we have chop, that was a thing for the older multithreaded compilers I think. You can use -subdivide to control the subdivision size for lightmapping; a value of zero disables subdivision entirely.

**Comment by JafiMapping** [comment]
Because i've been working for years with Knightmare Qbsp and it had this option to improve performance . Qbism Q2 tools 220 had it too.  I was mapping for  the Quake XP , and with all these dynamics lightning and the old Quake way to render triangles,  it  was very much an issue. I switched my project for the rerelease , and i guess its not an issue anymore, the rendering looks far more better.  In th e good old time you had to keep the r_speeds below 2500

Here is from the Knightmare website:
  To keep the r_speeds down in such huge expanses, maps compiled with QBSP -chop values of up to 1024 are supported.  This setting makes QBSP chop up surfaces every 1024 units, instead of every 240.  This setting is enabled by default in KMQBSP 1.13 and higher.  The reductions in r_speeds can be quite dramatic in some areas. 
![kmq2_polys_sm](https://github.com/ericwa/ericw-tools/assets/128496068/94443ecf-d0bf-4bd2-8938-1805e09a3b05)

**Comment by Paril** [comment, marked as answer]
Ah, yes. That's `-subdivide` in ericw-tools. A value of 0 disables chopping entirely.

**Reply by JafiMapping** [reply]
thank you! So The polycount should be dramatically lower right?

**Reply by Paril** [reply]
Depending on the size of the faces, it can reduce the polycount yea.
